### PR TITLE
Metro: Magnitude of WP 8.1 Update 2 parity fixes.

### PR DESCRIPTION
**Pull request by filiphsps**
~(Some whitespace fixes was accidentally also included, oh well).~

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,71 +1,71 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
```
Comment (louis993546) [inline build.gradle:1]: Note to self: expand #185 to covery any xml and gradle file as well

Comment (louis993546) [reply build.gradle:1]: @filiphsps can you revert file with just whitespace change?

File: demoAppDrawer/src/main/java/com/louis993546/metro/demo/appDrawer/DrawerPage.kt [hunk 2]
```diff
@@ -46,29 +54,41 @@ fun DrawerPage(
...
-    Row(modifier = modifier) {
-        SearchButton(modifier = Modifier.padding(all = topMargin)) { onAppClick(Apps.APP_SEARCH) }
+    Row(
+        modifier = modifier
+            .padding(horizontal = 6.dp)
+    ) {
+        SearchButton(
+            modifier = Modifier
+                .padding(horizontal = 12.dp)
+                .padding(vertical = 16.dp)
```
Comment (louis993546) [inline demoAppDrawer/src/main/java/com/louis993546/metro/demo/appDrawer/DrawerPage.kt:64]: nitpick: 
```suggestion
                .padding(horizontal = 12.dp, vertical = 16.dp),
```

**Comment by louis993546** [comment]
And @filiphsps do you have any doc/notes/references on WP7 or WP8? I just visually check with a 920 I bought to see if things are close enough, but if there are good doc on all the dimensions, parameters, or naming scheme, that'd be great!

**Comment by filiphsps** [comment]
> And @filiphsps do you have any doc/notes/references on WP7 or WP8? I just visually check with a 920 I bought to see if things are close enough, but if there are good doc on all the dimensions, parameters, or naming scheme, that'd be great!

Yes, I have about 20-30 Lumias laying in a box going from Windows Phone 7 through 7.8, 8, 8.1, 8.1 Update 2 and then a bunch of frankly unusable builds of Windows 10 Mobile.

When it comes to the general naming I figure sticking as close to the old XAML & silverlight documentation would be the best and simplest idea

**Comment by louis993546** [comment]
> Yes, I have about 20-30 Lumias laying in a box

Nice!


> I figure sticking as close to the old XAML & silverlight documentation

And naming wise, for `metro`, as it should be a lib that can be use by other people, IMO it should have more Android-friend names to make it easier for people to get started. Cause tbd, most people these days probably never know what those components were called back in the days. We can still add some simple alias-ish thing

For other modules, everything works 👍

**Comment by filiphsps** [comment]
> And naming wise, for `metro`, as it should be a lib that can be use by other people, IMO it should have more Android-friend names to make it easier for people to get started. Cause tbd, most people these days probably never know what those components were called back in the days. We can still add some simple alias-ish thing

Absolutely, I agree with that.

On a half-unrelated note; would it be okay to switch from using merge rebasing so we keep the commit history (I often use it to go back and read up on things I or someone else did in the past to get a better understanding for the codebase). It's a good habit I picked up from the SerenityOS codebase.

**Pull request merged**